packages/components/src/Form/Inputs/InputColor/InputColor.test.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import 'jest-styled-components'
+import '@testing-library/jest-dom/extend-expect'
+import React, { FormEvent, useState } from 'react'
+import { fireEvent } from '@testing-library/react'
+import {
+  assertSnapshotShallow,
+  renderWithTheme,
+} from '@looker/components-test-utils'
+
+import { Button } from '../../../Button'
+import { InputColor } from './InputColor'
+
+describe('InputColor', () => {
+  test('Default render', () => {
+    assertSnapshotShallow(<InputColor />)
```

Comment:
Same issue with this snapshot as the one in `FieldColor.test.tsx` – it's only an empty `ShallowWrapper`. Instead you could try this:
```suggestion
    const { queryByDisplayValue } = renderWithTheme(
      <InputColor value="yellow" />
    )
    expect(queryByDisplayValue('yellow')).toBeInTheDocument()
```